recipes-xfce/xfce-nilrt-settings/xfce-nilrt-settings_1.0.bb

```diff
@@ -65,6 +66,7 @@ FILES_${PN} = "${confdir}/autostart/dpms.desktop \
 	    ${confdir}/xfce4/xfconf/xfce-perchannel-xml/xfwm4.xml \
 	    ${confdir}/xfce4/xfconf/xfce-perchannel-xml/xsettings.xml \
 	    /usr/local/natinst/bin/dual-monitor-setup.sh \
+	    /etc/default/xserver-xfce \
```

Comment:
Here's another [recipe](https://github.com/ni/meta-nilrt/blob/nilrt/master/sumo/recipes-xfce/xserver-xfce-init/xserver-xfce-init_1.0.bb#L27) that adds this file. I think you can merge the contents of this default file to that file. I think that would be the part where we read from the nirtcfg settings to check if UI is enabled.